code/script.js

```diff
@@ -253,33 +254,48 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
+  board.style.display = 'flex';
+  winOrLose.style.display = 'none';
   // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
```

Comment:
It seams like you get an error in the console when the page is loaded and no question is selected yet.

Uncaught TypeError: Cannot read property 'value' of undefined
    at HTMLButtonElement.checkQuestion (script.js:307)